src/main/java/interpreter/Parser.java

```diff
@@ -176,6 +177,53 @@ private static String encodeQueuedTasks(Task task) {
         return queuedTaskString.toString();
     }
 
+
+    /**
+     * Encodes the Receipt for Storage.
+     * FORMAT: (commandType) (cashSpent) (date) /(tags)
+     * @param receipt Receipt object
+     * @return String to be stored/saved
+     */
+    public static String encodeReceipt(Receipt receipt) {
+        StringBuilder strSave = new StringBuilder();
+        strSave.append(encodeMainReceipt(receipt));
+        strSave.append("\n");
+        return strSave.toString();
+    }
```

Comment:
Do you really need this method? You could probably just append "/n" to the string in encodeMainReceipt or use the writer.append(feature).